packages/trpc/server/routers/viewer.tsx

```diff
@@ -159,7 +159,21 @@ const publicViewerRouter = router({
     }),
   // REVIEW: This router is part of both the public and private viewer router?
   slots: slotsRouter,
-  cityTimezones: publicProcedure.query(() => cityMapping),
+  cityTimezones: publicProcedure.query(() => {
+    const maxPops: { [key: string]: { city: string; timezone: string; pop: number } } = {};
+    cityMapping.forEach((city) => {
+      const cityPop = city.pop;
+      if (maxPops[city.city]?.pop === undefined || cityPop > maxPops[city.city].pop) {
+        maxPops[city.city] = { city: city.city, timezone: city.timezone, pop: city.pop };
+      }
```

Comment:
Can we add some explainers on what's happening here? It is kinda difficult to tell what's going on in the code.

Reply:
We are filtering out all cities that have the same "city" key and only using the one with the highest population. This way we return a new array of cities without running the risk of having more than one city with the same name on the dropdown and prevent users from mistaking the time zone of the desired city

Reply:
Can we put that in the code comments?